src/components/TimeseriesChart/stories/full.md

```diff
@@ -0,0 +1,45 @@
+# Timeseries Chart
+
+<!-- STORY -->
+
+### Description:
+
+This component loads the datapoints given a timeseries id and renders a line chart of those points.
+The component requires only `timeseriesIds`
+
+####
```

Comment:
Guess word **Usage** is should be here )

Reply:
Fixed